Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Subject: Drop rustc-rayon feature

I believe the rustc-rayon feature is only useful for building the
rustc compiler (it was added in indexmap 1.7.1), and my understanding
is that rustc in debian is *not* built from debcargo-produced crates.

So rather than intrdoucing an additional feature-related binary
package, which would cause a trip through the NEW queue, i'm just
stripping it out.

Another approach that avoids the new binary package, if it turns out
that stripping this feature causes problems, would be to add
collapse_features=true to debian/debcargo.toml.


--- indexmap/Cargo.toml
+++ indexmap/Cargo.toml
@@ -56,4 +56,0 @@
-[dependencies.rustc-rayon]
-version = "0.5"
-optional = true
-package = "rustc-rayon"
